frontend/src/metabase/entities/actions/actions.ts

```diff
@@ -57,11 +57,14 @@ const defaultImplicitActionCreateOptions = {
 const enableImplicitActionsForModel =
   async (modelId: number, options = defaultImplicitActionCreateOptions) =>
   async (dispatch: Dispatch) => {
-    if (options.insert) {
+    // We're ordering actions that's most recently created first.
```

Comment:
There is also a detail explanation in the [PR description here](https://github.com/metabase/metabase/pull/31184#issue-1733552826).